src/index.js

```diff
@@ -20,7 +21,18 @@ class DelegatedPeerRouting {
     this.dht = dht(this.api)
   }
 
+  /**
+   * Attempts to find the given peer
+   *
+   * @param {PeerID} id
+   * @param {function(Error, PeerInfo)} callback
+   * @returns {void}
+   */
   findPeer (id, callback) {
+    if (typeof id !== 'string') {
```

Comment:
I think it is a more correct validation to check if the `instance` is from `PeerID`, in order to guarantee that the `toB58String` function also exists.

```js
if (id instanceof PeerId) { ... }
````

What do you think @jacobheun ?